src/main.js

```diff
@@ -10,12 +10,30 @@ const PORT = 9000;
 const schemaValidator = new SchemaValidator(EQ_JSON_SCHEMA);
 const converter = new Convert(schemaValidator);
 
-app.get('/mock/graphql/:questionnaireId(\\d+)', (req, res) => {
-    res.json(GraphQLApi.getAuthorData(req.params['questionnaireId']));
+app.get('/mock/graphql/:questionnaireId(\\d+)', async (req, res, next) => {
+    try {
+        const result = await GraphQLApi.getAuthorData(req.params['questionnaireId']);
```

Comment:
could you add the `engines` field to package.json to document that it's only supported in version `7.10.0` of node and later